tls.go

```diff
@@ -11,6 +11,13 @@ import (
 
 func loadTLSConfig(settings *SessionSettings) (tlsConfig *tls.Config, err error) {
 	if !settings.HasSetting(config.SocketPrivateKeyFile) && !settings.HasSetting(config.SocketCertificateFile) {
+		if settings.HasSetting(config.SocketTLSForTesting) {
+			if forTesting, err := settings.BoolSetting(config.SocketTLSForTesting); err == nil {
+				return &tls.Config{InsecureSkipVerify: true}, nil
+			} else if forTesting {
```

Comment:
If err is not nil an error should be returned no matter what the value of forTesting as forTesting may be compromised.